### PR TITLE
[Turbo] Fixing support for not using old ClassUtils

### LINKED ISSUE #1469: Upgrade to Doctrine ORM 3.0 breaks Turbo
I tested upgrading to Doctrine ORM 3.0/DBAL 4.0 and ran into a problem where those packages no longer require `doctrine/common` but `ux-turbo` relies on the `ClassUtils` class from `common`.

I was able to fix the problem by directly requiring `doctrine/common` in my project, but I think `ux-turbo` should require `doctrine/common` unless there are plans to move away from using the `ClassUtils` class (I may have missed discussion about this somewhere).

### PR DESCRIPTION
**Pull request by weaverryan**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Issues        | Fix #1469
| License       | MIT

### CODE REVIEW

**Comment by weaverryan** [comment]
This, indeed, fixes the reported problem

File: src/Turbo/src/Doctrine/ClassUtil.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            $entityClass = get_parent_class($entity);
+            if (false === $entityClass) {
+                throw new \LogicException('Parent class missing');
+            }
+
+            return $entityClass;
+        }
+
+        // @legacy for old versions of Doctrine
+        if (class_exists(ClassUtils::class)) {
```
Comment (stloyd) [inline src/Turbo/src/Doctrine/ClassUtil.php:33]: This class is not imported so it will always return false.

Comment (jrushlow) [reply src/Turbo/src/Doctrine/ClassUtil.php:33]: https://github.com/symfony/ux/pull/1475

Comment (jrushlow) [reply src/Turbo/src/Doctrine/ClassUtil.php:33]: Good catch!

File: src/Turbo/src/Doctrine/ClassUtil.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ * @internal
+ */
+final class ClassUtil
+{
+    public static function getEntityClass(object $entity): string
+    {
+        if ($entity instanceof LazyObjectInterface) {
+            $entityClass = get_parent_class($entity);
+            if (false === $entityClass) {
+                throw new \LogicException('Parent class missing');
```
Comment (nicolas-grekas) [inline src/Turbo/src/Doctrine/ClassUtil.php:26]: no need for this exception IMHO, using get_class should be the way if there's no parent

File: src/Turbo/src/Doctrine/ClassUtil.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+/**
+ * @internal
+ */
+final class ClassUtil
+{
+    public static function getEntityClass(object $entity): string
+    {
+        if ($entity instanceof LazyObjectInterface) {
+            $entityClass = get_parent_class($entity);
```
Comment (nicolas-grekas) [inline src/Turbo/src/Doctrine/ClassUtil.php:24]: `return get_parent_class($entity) ?: \get_class($entity);`

**Pull request merged**